cli/compile.js

```diff
@@ -0,0 +1,33 @@
+const webpack = require('webpack');
+const ProgressBarPlugin = require('progress-bar-webpack-plugin');
+
+function main(config) {
+    const compiler = webpack(config);
+
+    new ProgressBarPlugin({
+        format: '  build [:bar] :percent (:elapsed seconds)',
+        clear: false,
+        width: 60
+    }).apply(compiler);
+
+    compiler.run(function(err, stats) {
+        if (err) {
+            console.error(err.stack || err);
+            if (err.details) {
+                console.error(err.details);
+            }
+            return;
```

Comment:
why not an exit here ?
PS for errors you can use https://github.com/ProtonMail/proton-pack/blob/90deb3e3e2694dd0bfea4d3a6e8d4775e660282f/cli/log.js#L27 (_so we always have the same output for them_)